MLS.Agent.Tests/CommandLine/CommandLineParserTests.cs

```diff
@@ -214,6 +214,20 @@ public async Task AiKey_defaults_to_null()
             _start_options.ApplicationInsightsKey.Should().BeNull();
         }
 
+        [Fact]
+        public async Task Parses_the_port()
+        {
+            await _parser.InvokeAsync("--port 6000", _console);
+            _start_options.Port.Should().Be(6000);
+        }
+
+        [Fact]
+        public async Task Takes_5000_as_the_default_port()
+        {
+            await _parser.InvokeAsync("", _console);
+            _start_options.Port.Should().Be(5000);
```

Comment:
We might want to consider changing this from the default Kestrel ports. 

Also keep in mind we also listen on 5001 by default for https.